src/governance/Migrate.sol

```diff
@@ -14,28 +14,28 @@ contract Migrate is Math {
         mapping (uint => bool) voted; // cdp => voted
     }
 
-    Timelock public timelock;
+    uint public constant DELAY = 2 days;
+
     JarConnector public jarConnector;
     BCdpManager public man;
+    GovernanceExecutor public executor;
 
     Proposal[] public proposals;
 
     constructor(
         JarConnector jarConnector_,
-        BCdpManager man_
+        BCdpManager man_,
+        GovernanceExecutor executor_
     ) public {
         jarConnector = jarConnector_;
         man = man_;
-    }
-
-    function setTimelock(Timelock timelock_) external {
-        require(timelock == Timelock(0), "timelock-already-set");
-        timelock = timelock_;
+        executor = executor_;
     }
 
     function propose(address newOwner) external returns (uint) {
```

Comment:
we need events for propose, vote, cancel vote and probably queue and execute as well.